src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    /** Scanner to scan in user input */
+    private Scanner sc;
+
+    /**
+     * Constructor for a UI.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    };
+
+    /**
+     * UI when a task has been added.
```

Comment:
> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)

Would it be better this way?
```suggestion
     * Displays UI to show a task has been added.
```

I noticed the same issue in several other places within this file.